bip-xxxx.mediawiki

```diff
@@ -0,0 +1,62 @@
+<pre>
+  BIP: ?
+  Layer: Applications
+  Title: unspendable() Descriptor Key Expression
+  Author: Andrew Toth <andrewstoth@gmail.com>
+          Kewde <kewde@particl.io>
+  Comments-Summary: No comments yet.
+  Comments-URI: https://github.com/bitcoin/bips/wiki/Comments:BIP-?
+  Status: Draft
+  Type: Standards Track
+  Created: 2024-12-12
+  License: BSD-2-Clause
+</pre>
+
+==Abstract==
+This document specifies a <tt>unspendable()</tt> key expression for output script descriptors. The <tt>unspendable()</tt> expression takes multiple public keys as input and produces an unspendable public key that can be independently verified by anyone with knowledge of all the constituent public keys.
+
+==Copyright==
+
+This BIP is licensed under the BSD 2-clause license.
+
+==Motivation==
+
+This document introduces a mechanism to compute a NUMS (Nothing Up My Sleeve) point for use in the Taproot key path that:
+* Allows active participants involved in constructing the output script to independently verify the unspendable key.
+* Prevents passive observers from recognizing that the key path is unspendable.
+* Enables signers with limited information, such as hardware wallets, to verify unspendability without requiring user interaction.
```

Comment:
Another motivation is having a deterministic key that can be recreated from the rest of the descriptor, so that the entropy does not have to be backed up separately.